server/routing/game.go

```diff
@@ -13,27 +18,54 @@ const (
 
 // GameRoutes contains information about routes specific to game interaction
 type GameRoutes struct {
-	Service services.GameService
+	Services *services.MasterList
```

Comment:
I'm not sure I understand what is happening here?  What is the MasterList vs GameService?

Reply:
MasterList is just a struct that has the GameService and a LobbyService so far

Reply:
Yeah, this used to only hold a *GameService pointer, now it just has the whole *MasterList object (which has Game Service as a member). I was just making it similar to how the other route structs look.